### PR TITLE
BXMSDOC-8424: Add Infinispan content and correct section

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

**Review by msmagnanijr — APPROVED** [review]
lgtm

File: doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc [hunk 1]
```diff
@@ -1022,12 +1022,112 @@ $ {PRODUCT_INIT} deploy-service example-quarkus https://github.com/kiegroup/kogi
...
 === Integrating a custom {PRODUCT} service with Infinispan
 
-<Start here>
-////
+You can integrate a custom {PRODUCT} service with an infinispan cluster using the following configuration modes:
+
+* Automatic configuration
+* Manual configuration
+
+==== Integrating a custom {PRODUCT} service with Infinispan using automatic configuration mode
```
Comment (kaldesai) [inline doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc:1035]:
```suggestion
==== Integrating a custom {PRODUCT} service with an infinispan using automatic configuration mode
```

File: doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc [hunk 2]
```diff
@@ -1022,12 +1022,112 @@ $ {PRODUCT_INIT} deploy-service example-quarkus https://github.com/kiegroup/kogi
...
+
+In automatic configuration mode the {PRODUCT} operator automatically fetches the relevant infinispan connection details from the Infinispan custom resource and provides it to the custom {PRODUCT} service.
+
+The Infinispan Operator provides operational intelligence and reduces management complexity for deploying Infinispan on Kubernetes and {OPENSHIFT}.
```
Comment (kaldesai) [inline doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc:1039]:
```suggestion
The Infinispan operator provides operational intelligence and reduces management complexity for deploying Infinispan on Kubernetes and {OPENSHIFT}.
```

File: doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc [hunk 3]
```diff
@@ -1022,12 +1022,112 @@ $ {PRODUCT_INIT} deploy-service example-quarkus https://github.com/kiegroup/kogi
...
+
+For more information about installing Infinispan Operator and setting up an Infinispan cluster, see the https://infinispan.org/docs/infinispan-operator/main/operator.html[Infinispan Operator 2.2 Guide].
```
Comment (kaldesai) [inline doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc:1041]:
```suggestion
For more information about installing Infinispan operator and setting up an Infinispan cluster, see the https://infinispan.org/docs/infinispan-operator/main/operator.html[Infinispan Operator 2.2 Guide].
```

File: doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc [hunk 4]
```diff
@@ -1022,12 +1022,112 @@ $ {PRODUCT_INIT} deploy-service example-quarkus https://github.com/kiegroup/kogi
...
+  name: kogito-infinispan-infra
+spec:
+  resource:
+    apiVersion: infinispan.org/v1
+    kind: Infinispan
+    name: kogito-infinispan
+    namespace: infinispan-cluster-namespace
+----
+
+. To enable Infinispan persistence, bind KogitoInfra with the {PRODUCT} Service by doing one of the following tasks:
```
Comment (kaldesai) [inline doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc:1070]:
```suggestion
. To enable Infinispan persistence, bind KogitoInfra with the {PRODUCT} service by performing one of the following tasks:
```

File: doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc [hunk 5]
```diff
@@ -1022,12 +1022,112 @@ $ {PRODUCT_INIT} deploy-service example-quarkus https://github.com/kiegroup/kogi
...
+metadata:
+  name: travels
+spec:
+  infra:
+    - kogito-infinispan-infra
+----
+
+==== Integrating a custom {PRODUCT} service with Infinispan using manual configurations
+
+If you set up a custom Infinispan cluster, you can refer to it in the {PRODUCT}Runtime custom resource by configuring the following environment variables and application properties:
```
Comment (kaldesai) [inline doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc:1094]:
```suggestion
If you set up a custom Infinispan cluster, you can refer to it in the {PRODUCT} Runtime custom resource by configuring the following environment variables and application properties:
```

File: doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc [hunk 6]
```diff
@@ -1022,12 +1022,112 @@ $ {PRODUCT_INIT} deploy-service example-quarkus https://github.com/kiegroup/kogi
...
 
 <Start here>
+////
 
 [id="proc-custom-kogito-service-integration-infinispan_{context}"]
 === Integrating a custom {PRODUCT} service with Infinispan
 
-<Start here>
-////
+You can integrate a custom {PRODUCT} service with an infinispan cluster using the following configuration modes:
```
Comment (kaldesai) [inline doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc:1030]:
```suggestion
You can integrate a custom {PRODUCT} service with the Infinispan cluster using the following configuration modes:
```

**Review by kaldesai — APPROVED** [review]
Hey @csherrar, I have added minor suggestions. Otherwise your content looks good to me. Very well written! Thank you! :)

File: doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc (hunk 3 above)
Comment (csherrar) [reply doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc:1041]: Done

File: doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc (hunk 5 above)
Comment (csherrar) [reply doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc:1094]: Done

File: doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc (hunk 6 above)
Comment (csherrar) [reply doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc:1030]: Done

File: doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc (hunk 2 above)
Comment (csherrar) [reply doc-content/kogito-docs/src/main/asciidoc/openshift/chap-kogito-deploying-on-openshift.adoc:1039]: Done

**Pull request merged**